public/components/tooltip/display_features.tsx

```diff
@@ -32,6 +32,10 @@ export const DisplayFeatures = memo(({ map, layers }: Props) => {
 
       const features = map.queryRenderedFeatures(e.point);
       if (features && map) {
+        // don't show tooltip from labels
+        if (features.length === 1 && features[0].layer?.type === 'symbol') {
```

Comment:
what happens if there are more than one features from multiple layers?

Reply:
If there are multiple features, which means the features on this point not only has label, the if check will be ignored.